Extensions/TeamCity/Src/vss-extension.json

```diff
@@ -73,7 +73,7 @@
           },
           {
             "name": "Builds",
-            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds",
+            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds/?locator=branch:default:any",
```

Comment:
@GitHubSriramB Sorry, I don't understand what you mean. While linking to an artifact from this extension no builds are listed on the "Link an artifact source" (There's no default version or similar). And to have builds from all branches available in the "Create new release" dialog was the goal of this PR.